src/App.js

```diff
@@ -1,8 +1,76 @@
-function App() {
+import { useState } from "react";
+import sunny from "./sunny.png"
+import rainy from "./rainy.png"
+import semi from "./semi.png"
+import cloudy from "./cloudy.png"
+
+var picture= sunny;
```

Comment:
In almost all cases, we use 'const' in React; 'var' is rarely used. This is because you can't reassign a variable if you declare it using 'const', which can save you from bugs. So please redeclare all your variables in the app using 'const'